src/bin/tree-sitter-grep.rs

```diff
@@ -1,11 +1,11 @@
 use std::process;
 
 use clap::Parser;
-use tree_sitter_grep::{run, Args, RunStatus};
+use tree_sitter_grep::{run_print, Args, RunStatus};
 
 pub fn main() {
     let args = Args::parse();
-    match run(args) {
+    match run_print(args) {
```

Comment:
Naming?

Reply:
(And that comment is related to naming because having to come up with various `run_something` names seems like the result of a weird split between the bin and lib to me.)

Reply:
Ya agreed I think we're a little bit "weird-shaped"

I'm personally probably ok with the crate-level API feeling a bit like "hey yes this is hacked-onto something that is most naturally used via the command line"

That being said I think instantiating `Args` the way it's done here is by far _too_ hacked-on